### PR TITLE
OSGI Tasks Implementations

### PR DESCRIPTION
**Pull request by kirilAtan**
Sample implementations of the [first part of the OSGi Coding Tasks](http://docs.openhab.org/developers/prerequisites/osgitasks.html#i-writing-basic-osgi-bundle).

This is related to https://github.com/openhab/openhab-docs/issues/85

### CODE REVIEW

**Comment by ThomDietrich** [comment]
Hey @kirilAtan,
Nice work! Are you going to reference the example code in any of the docs articles?

Is the base folder with its long name and the spaces a good idea? I'm a bit old fashioned related to things line this :laughing:

**Comment by kirilAtan** [comment]
I hadn't thought of that, but that is a great idea to reference the example code. I will speak with the author to add links to the implementations when they are committed and approved. 
Yes, you are right. This is definitely not a suitable name.
In the next commit I would like to try a different approach - I will put all the bundles in one folder. If there is a specific folder for every task it becomes very complicated, because almost every bundle imports bundles from other tasks. A bundle from the last task needs bundles from four other tasks. If it isn't suitable the bundles can always be grouped in folders again.

**Comment by kirilAtan** [comment]
I've moved all the bundles to a common folder named bundles. 
If there was a different folder for every task then to run a bundle from the last task one must import bundles from four other tasks(folders).

**Comment by kirilAtan** [comment]
I've moved all the bundles to a common folder named bundles. 
If there was a different folder for every task then to run a bundle from the last task one must import bundles from four other tasks(folders).

File: _sample_code/osgi_codings_tasks/bundles/org.openhab.training.helloosgi.modified/about.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1"/>
+<title>About</title>
+</head>
+<body lang="EN-US">
+<h2>About This Content</h2>
+
+<p>&lt;<em>September 15, 2014</em>&gt;</p>
+<h3>License</h3>
+
+<p>The Eclipse Foundation makes available all content in this plug-in (&quot;Content&quot;).  Unless otherwise
```
Comment (kaikreuzer) [inline _sample_code/osgi_codings_tasks/bundles/org.openhab.training.helloosgi.modified/about.html]: Reading this, it reminds me that we need to rework this text - "The Eclipse Foundation" is not involved in any way here... What about saying "The openHAB community"?

File: _sample_code/osgi_codings_tasks/bundles/org.openhab.training.helloosgi.modified/about.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+indicated below, the Content is provided to you under the terms and conditions of the
+Eclipse Public License Version 1.0 (&quot;EPL&quot;).  A copy of the EPL is available
+at <a href="http://www.eclipse.org/legal/epl-v10.html">http://www.eclipse.org/legal/epl-v10.html</a>.
+For purposes of the EPL, &quot;Program&quot; will mean the Content.</p>
+
+<p>If you did not receive this Content directly from the Eclipse Foundation, the Content is
```
Comment (kaikreuzer) [inline _sample_code/osgi_codings_tasks/bundles/org.openhab.training.helloosgi.modified/about.html]: same here.

File: _sample_code/osgi_codings_tasks/bundles/org.openhab.training.helloosgi.modified/about.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+being redistributed by another party (&quot;Redistributor&quot;) and different terms and conditions may
+apply to your use of any object code in the Content.  Check the Redistributor's license that was
+provided with the Content.  If no such license exists, contact the Redistributor.  Unless otherwise
+indicated below, the terms and conditions of the EPL still apply to any source code in the Content
+and such source code may be obtained at <a href="http://www.eclipse.org/">http://www.eclipse.org</a>.</p>
```
Comment (kaikreuzer) [inline _sample_code/osgi_codings_tasks/bundles/org.openhab.training.helloosgi.modified/about.html]: may be obtained at openhab.org

**Comment by kirilAtan** [comment]
I've updated the about.html files:
https://github.com/openhab/openhab2-addons/pull/1263

**Comment by kaikreuzer** [comment]
Thanks a lot!

**Pull request merged**